python/grass/pygrass/vector/__init__.py

```diff
@@ -450,6 +450,7 @@ def viter(self, vtype, idonly=False):
 
             >>> test_vect.close()
         """
+        is2D = bool(libvect.Vect_is_3d(self.c_mapinfo) != 1)
```

Comment:
The same code is in the Geo class. What put me on that trail was that slightly better code would be just `not bool(Vect_is_3d(...)`, but my conclusion now it that for this PR, this line is simply not needed.

Reply:
See my comments below. I changed this line to call `is_3D()` similarly as it's done here: https://github.com/OSGeo/grass/blob/main/python/grass/pygrass/vector/__init__.py#L518

Reply:
Much simpler!